input/pagecontent/3-1-2-eventmaturitymodel.md

```diff
@@ -6,7 +6,7 @@ Maturity Level | Maturity title | Requirements
 1 | Submitted  | _The above, and …_ Event definition is written up as a pull request using the [Event template](3-1-1-template.html) and community feedback is solicited from the community (e.g. the zulip FHIRcast stream](https://chat.fhir.org/#narrow/stream/179271-FHIRcast)).
 2 | Tested | _The above, and …_ The event has been tested and successfully supports interoperability among at least one Hub and two independent subscribing apps using semi-realistic data and scenarios (e.g. at a FHIR Connectathon). The github pull request defining the event is approved and published.
 3 | Considered |  _The above, and …_ At least 3 distinct organizations recorded ten distinct implementer comments (including a github or jira issue, tracker item, or comment on the event definition page), including at least two Hubs and three subscribing apps. The event has been tested at two Connectathons.
-4 | Documented | _The above, and …_ The author agrees that the artifact is sufficiently stable to require implementer consultation for subsequent non-backward compatible changes.  The event is implemented in the standard FHIRcast reference implementation and multiple prototype projects. The Event specification SHALL: <ul><ol>Identify a broad set of example contexts in which the event may be used with a minimum of three, but as many as 10.</ol><ol>Clearly differentiate the event from similar events or other standards to help an implementer determine if the event is correct for their scenario.</ol><ol>Explicitly document example scenarios when the event should not be used.</ol></ul>
+4 | Documented | _The above, and …_ The author agrees that the artifact is sufficiently stable to require implementer consultation for subsequent non-backward compatible changes.  The event is implemented in the standard FHIRcast reference implementation and multiple prototype projects. The Event specification SHALL: <ol>Identify a broad set of example contexts in which the event may be used with a minimum of three, but as many as 10. </ol> <ol>Clearly differentiate the event from similar events or other standards to help an implementer determine if the event is correct for their scenario.</ol> <ol>Explicitly document example scenarios when the event should not be used.</ol>
```

Comment:
```suggestion
4 | Documented | _The above, and …_ The author agrees that the artifact is sufficiently stable to require implementer consultation for subsequent non-backward compatible changes.  The event is implemented in the standard FHIRcast reference implementation and multiple prototype projects. The Event specification SHALL: <ul><li>Identify a broad set of example contexts in which the event may be used with a minimum of three, but as many as 10.</li><li>Clearly differentiate the event from similar events or other standards to help an implementer determine if the event is correct for their scenario.</li> <li>Explicitly document example scenarios when the event should not be used.</li> </ul>
```
ol or ul creates the list, li specifies each line in the list. Going with unordered, because this content already exists within an ordered list.